### PR TITLE
6245: Changing the footer for identity

### PR DESCRIPTION
**Pull request by RuthBerry96**

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Identity.Api/Styles/main.scss [hunk 1]
```diff
@@ -5,4 +5,8 @@
 @import 'node_modules/buying-catalogue-components/app/styles/main';
 
 // Buying catalogue login page styles
-@import 'login';
\ No newline at end of file
+@import 'login';
+
+.bc-t-bg--grey-4 {
+    background-color: #d8dde0
```
Comment (kat1906) [inline src/NHSD.BuyingCatalogue.Identity.Api/Styles/main.scss:11]: please use nhs colours - https://service-manual.nhs.uk/design-system/styles/colour

File: src/NHSD.BuyingCatalogue.Identity.Api/Styles/main.scss [hunk 2]
```diff
@@ -5,4 +5,8 @@
...
+}
```
Comment (kat1906) [inline src/NHSD.BuyingCatalogue.Identity.Api/Styles/main.scss:12]: github likes a new line at the end of the file

File: src/NHSD.BuyingCatalogue.Identity.Api/Styles/main.scss (hunk 1 above)
Comment (kat1906) [reply src/NHSD.BuyingCatalogue.Identity.Api/Styles/main.scss:11]: $color_nhsuk-grey-4

File: src/NHSD.BuyingCatalogue.Identity.Api/Views/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -89,24 +89,35 @@
...
+                    <span class='nhsuk-u-margin-right-1'>Legal</span>
+                    <span class='nhsuk-u-margin-right-1'><a href='https://digital.nhs.uk/about-nhs-digital/privacy-and-cookies'>Privacy and Cookies</a></span>
+                </div>
             </div>
         </div>
-    </footer>
+    </div>
+
 </body>
 </html>
```
Comment (kat1906) [inline src/NHSD.BuyingCatalogue.Identity.Api/Views/Shared/_Layout.cshtml:123]: new line

File: src/NHSD.BuyingCatalogue.Identity.Api/Styles/main.scss (hunk 1 above)
Comment (RuthBerry96) [reply src/NHSD.BuyingCatalogue.Identity.Api/Styles/main.scss:11]: Is it better using 
background-color: $nhsuk-border-color 

or do you want
background-color: $color_nhsuk-grey-4

Comment (kat1906) [reply src/NHSD.BuyingCatalogue.Identity.Api/Styles/main.scss:11]: I prefer this as it is explicit:  $color_nhsuk-grey-4

**Pull request merged**